### PR TITLE
boards: nxp: Update board documentation of RW612

### PR DESCRIPTION
**Pull request by MaochenWang1**
Add information about Wi-Fi.

### CODE REVIEW

**Comment by MaochenWang1** [comment]
Hi @dleach02  @decsny please review

File: boards/nxp/frdm_rw612/doc/index.rst [hunk 1]
```diff
@@ -36,8 +36,6 @@ Supported Features
 +-----------+------------+-----------------------------------+
 | USART     | on-chip    | serial                            |
 +-----------+------------+-----------------------------------+
-| BLE       | on-chip    | Bluetooth                         |
```
Comment (decsny) [inline boards/nxp/frdm_rw612/doc/index.rst:39]: why is this deleted ?

**Review by decsny — CHANGES_REQUESTED** [review]

File: boards/nxp/frdm_rw612/doc/index.rst (hunk 1 above)
Comment (MaochenWang1) [reply boards/nxp/frdm_rw612/doc/index.rst:39]: it’s duplicated, you can expand the hidden lines

Comment (decsny) [reply boards/nxp/frdm_rw612/doc/index.rst:39]: ok, you can mention this in commit message

**Review by decsny — APPROVED** [review]

**Review by mmahadevan108 — APPROVED** [review]

**Review by dleach02 — APPROVED** [review]
